.github/workflows/go.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-go@v3
       with:
-        go-version: '1.19'
+        go-version: '1.21.0'
```

Comment:
Should we use a matrix strategy execution to test against 1.20.7 (last release of a supported version) at the same time?

Reply:
When we start using the latest built-ins & packages of Go 1.21, tests will fail against anything lower than go version 1.21. Eventually, we can only test against the minimum required version of Go i.e. >= 1.21.